Surface_mesh_segmentation/examples/Surface_mesh_segmentation/sdf_values_example.cpp

```diff
@@ -17,7 +17,7 @@ int main()
   Polyhedron mesh;
   std::ifstream input("data/cactus.off");
   if ( !input || !(input >> mesh) || mesh.empty() || ( !CGAL::is_triangle_mesh(mesh)) ) {
-    std::cerr << "Input is not a triangle mesh" << std::endl;
+    std::cout << "Input is not a triangle mesh" << std::endl;
```

Comment:
When we discussed it I though you agreed that cerr is useful if  cin and cout are used for executables that are used with redirection and pipes, but that in the examples we can use cout for error output as well as for output that reports #vertices, time, etc.